src/reference/asciidoc/mqtt.adoc

```diff
@@ -164,6 +164,9 @@ StaticMessageHeaderAccessor.acknowledgment(someMessage).acknowledge();
 ----
 ====
 
+Starting with version `5.2.11`, when message converter throws an exception or returns `null` from the `MqttMessage` conversion, the `MqttPahoMessageDrivenChannelAdapter` sends an `ErrorMessage` into the provided `errorChannel`.
+Re-throws this conversion error otherwise into an MQTT client callback.
```

Comment:
```suggestion
Starting with version `5.2.11`, when the message converter throws an exception or returns `null` from the `MqttMessage` conversion, the `MqttPahoMessageDrivenChannelAdapter` sends an `ErrorMessage` into the `errorChannel`, if provided.
Re-throws this conversion error otherwise into an MQTT client callback.
```

Also see my comment in the code about re-throwing.